### PR TITLE
✨Better command line processing in e2e-conformance.sh

### PR DESCRIPTION
**Pull request by dims**
Allow stuff to be passed in as command line params

Also switch on the flag to use latest CI artifacts

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: scripts/ci-conformance.sh [hunk 1]
```diff
@@ -84,7 +84,7 @@ if grep -iqF "$(echo "${AWS_ACCESS_KEY_ID-}" | \
 fi
 
 # TODO(dims): remove "SKIP_INIT_IMAGE" once we fix problem building images in CI
-SKIP_INIT_IMAGE=yes hack/ci/e2e-conformance.sh --verbose
+hack/ci/e2e-conformance.sh --verbose --skip-init-image --use-ci-artifacts
```
Comment (detiber) [inline scripts/ci-conformance.sh]: Wouldn't we want to limit the use of `--use-ci-artifacts` to just the "stable" job?

Comment (dims) [reply scripts/ci-conformance.sh]: @detiber y, there's more tweaking to do here. i want to be able to specify this in the CI job definition itself as well. For now, i want to enable this across all the jobs to see how it holds up over the weekend

**Pull request merged**